Vagrantfile

```diff
@@ -79,6 +79,10 @@ Vagrant.configure(2) do |config|
     # Install our own varnish config file
     sudo cp /vagrant/mapit.mysociety.org/conf/varnish.vcl-example /etc/varnish/default.vcl
     sudo service varnish restart
+
+    # Auto-activate virtualenv on login
+    echo >> /home/vagrant/.bashrc "source /vagrant/virtualenv-mapit/bin/activate"
```

Comment:
I think this should be `/vagrant/mapit.mysociety.org/.venv/bin/activate` if I've got the location right?
Also I see the mapit Vagrantfile installs py-inotify to speed up runserver so might as well do that here too.